Crash when InfluxDB url is invalid.

"missing colon in influx url"

```
C:\Users\user\Downloads\unifi-poller.amd64.exe>unifi-poller.amd64.exe -c up.conf
2022/01/20 19:32:02 [INFO] Loading Configuration File: up.conf
2022/01/20 19:32:02.065813 start.go:88: [DEBUG] Debug Logging Enabled
2022/01/20 19:32:02.065813 start.go:91: [INFO] UniFi Poller v2.1.3 Starting Up! PID: 6336
2022/01/20 19:32:02.068049 updateweb.go:213: [DEBUG] Requesting https://192.168.10.1/ to determine API paths
2022/01/20 19:32:02.154960 updateweb.go:213: [DEBUG] Using NEW UniFi controller API paths for https://192.168.10.1/
2022/01/20 19:32:02.154960 updateweb.go:213: [DEBUG] Requesting https://192.168.10.1/api/auth/login, with params: true, cookies: 0
2022/01/20 19:32:02.379418 updateweb.go:213: [DEBUG] Requested https://192.168.10.1/api/auth/login: elapsed 224ms, returned 3172 bytes
2022/01/20 19:32:02.380031 updateweb.go:213: [DEBUG] Requesting https://192.168.10.1/proxy/network/status, with params: false, cookies: 1
2022/01/20 19:32:02.432311 updateweb.go:213: [DEBUG] Requested https://192.168.10.1/proxy/network/status: elapsed 40ms, returned 76 bytes
2022/01/20 19:32:02.433111 updateweb.go:213: [DEBUG] Authenticated with controller successfully, https://192.168.10.1
2022/01/20 19:32:02.440253 updateweb.go:213: [DEBUG] Checking Controller Sites List
2022/01/20 19:32:02.441505 updateweb.go:213: [DEBUG] Requesting https://192.168.10.1/proxy/network/api/stat/sites, with params: false, cookies: 1
2022/01/20 19:32:02.646838 updateweb.go:213: [DEBUG] Requested https://192.168.10.1/proxy/network/api/stat/sites: elapsed 205ms, returned 1653 bytes
2022/01/20 19:32:02.648568 updateweb.go:213: [DEBUG] Found 1 site(s): default
2022/01/20 19:32:02.654748 updateweb.go:193: [INFO] Found 1 site(s) on controller https://192.168.10.1: default (Default)
2022/01/20 19:32:02.655964 updateweb.go:193: [INFO] Configured UniFi Controller 1 of 1:
2022/01/20 19:32:02.657570 updateweb.go:193: [INFO]    => URL: https://192.168.10.1 (verify SSL: false)
2022/01/20 19:32:02.657570 updateweb.go:193: [INFO]    => Version:  (a629a6b7-8e1e-5410-ba38-b7efde1e9eee)
2022/01/20 19:32:02.660689 updateweb.go:193: [INFO]    => Username: unifipoller (has password: true)
2022/01/20 19:32:02.660689 updateweb.go:193: [INFO]    => Hash PII / Poll Sites: false / all
2022/01/20 19:32:02.660689 updateweb.go:193: [INFO]    => Save Sites / Save DPI: true / false (metrics)
2022/01/20 19:32:02.660689 updateweb.go:193: [INFO]    => Save Events / Save IDS: false / false (logs)
2022/01/20 19:32:02.666426 updateweb.go:193: [INFO]    => Save Alarms / Anomalies: false / false (logs)
2022/01/20 19:32:02.667760 updateweb.go:193: [INFO]    => Save Rogue APs: false
2022/01/20 19:32:02.670102 logger.go:17: [INFO] Prometheus config missing (or disabled), Prometheus HTTP listener disabled!
panic: send on closed channel

goroutine 37 [running]:
github.com/unifi-poller/poller.(*UnifiPoller).runOutputMethods.func1(0xc000226600, 0xc0000598a0, 0xc000178720)
        /home/travis/gopath/pkg/mod/github.com/unifi-poller/poller@v0.0.0-20210418074507-efda34dd8222/outputs.go:85 +0x73
created by github.com/unifi-poller/poller.(*UnifiPoller).runOutputMethods
        /home/travis/gopath/pkg/mod/github.com/unifi-poller/poller@v0.0.0-20210418074507-efda34dd8222/outputs.go:84 +0xf8
```

--- a/pkg/influxunifi/influxdb.go
+++ b/pkg/influxunifi/influxdb.go
@@ -5,6 +5,7 @@ package influxunifi
 import (
 	"crypto/tls"
 	"fmt"
+	"net/url"
 	"os"
 	"strconv"
 	"strings"
@@ -154,6 +155,12 @@ func (u *InfluxUnifi) Run(c poller.Collect) error {
 	var err error
 
 	u.setConfigDefaults()
+	
+	_, err = url.Parse(u.Config.URL)
+	if err != nil {
+		u.LogErrorf("invalid influx URL: %v", err)
+		return err
+	}
 
 	if u.IsVersion2 {
 		// we're a version 2

